### PR TITLE
Add client name interpretation for new stats pipeline.

### PR DESCRIPTION
**Pull request by shishirx34**

### CODE REVIEW

**Comment by agr** [comment]
Could you please reference the original SQL scripts it has been ported from?

**Review by agr — APPROVED** [review]

File: src/Stats.LogInterpretation/ClientNameTranslation.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    public static class ClientNameTranslation
+    {
+        public static string GetClientCategory(string clientName)
+        {
+            if (string.IsNullOrWhiteSpace(clientName))
+            {
+                return string.Empty;
+            }
+
+            if (clientName.Contains(ClientNames.NuGet))
```
Comment (loic-sharma) [inline src/Stats.LogInterpretation/ClientNameTranslation.cs:17]: Should these be case insensitive checks?

File: src/Stats.LogInterpretation/ClientNames.cs [hunk 2]
```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stats.LogInterpretation
+{
+    public static class ClientNames
+    {
+        public static IList<string> NuGet = new List<string>()
```
Comment (loic-sharma) [inline src/Stats.LogInterpretation/ClientNames.cs:9]: Can these be readonly?

```suggestion
        public static readonly IReadOnlyList<string> NuGet = new List<string>
```

File: src/Stats.LogInterpretation/ClientNameTranslation.cs (hunk 1 above)
Comment (loic-sharma) [inline src/Stats.LogInterpretation/ClientNameTranslation.cs:17]: Without Intellisense it's hard to realize this is not doing a regular `string.Contains(string)` case sensitive check.

I'd consider making this a little more obvious. Maybe rename the method to `MatchesClientName` or something.

**Pull request merged**